src/components/huwelijksplanner/PersonalDataList.tsx

```diff
@@ -0,0 +1,82 @@
+import { DataList, DataListItem, DataListKey, DataListValue, NumberValue } from "@utrecht/component-library-react";
+import { useTranslation } from "next-i18next";
+import { IngeschrevenPersoon } from "../../generated";
+
+export const PersonalDataList = ({ partner }: { partner: IngeschrevenPersoon }) => {
+  const { t } = useTranslation(["common", "huwelijksplanner-step-4", "form"]);
+
+  return (
+    <DataList aria-describedby="personal-details" className="utrecht-data-list--rows">
+      <DataListItem>
+        <DataListKey>{t("form:bsn")}</DataListKey>
+        <DataListValue value={partner.burgerservicenummer ?? ""} emptyDescription={t("form:data-item-unknown")}>
```

Comment:
In the `AddressDataList.tsx` file the DataListValue.value doesn't have an explicit `""` fallback. I'm not sure if we need it but we might want it consistent?